### PR TITLE
Page structure Initialized

### PR DESCRIPTION
**Pull request by commcheck396**
Upload the newly updated blank page structure. All web pages development can base on that. #41

### CODE REVIEW

File: sidebar/sidebar.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+    .subtasks {
+      margin-left: 20px;
+      display: none;
+    }
+
+    .subtasks.visible {
+      display: block;
+    }
+  </style>
```
Comment (Earth34r) [inline sidebar/sidebar.html:66]: Maybe link external CSS here instead of inlining it?

Comment (commcheck396) [reply sidebar/sidebar.html:66]: yeah, will optimize it.

**Review by jain-atishay — APPROVED** [review]
It looks like it adds the feature of random quote we agree on as team and i believe external CSS file separation  is a good idea

**Comment by SirKanaad26** [comment]
Code invalidated.

**Pull request closed without merging**